sdk/src/types/block/core/mod.rs

```diff
@@ -21,6 +21,7 @@ pub use self::{
     validation::{ValidationBlock, ValidationBlockBuilder},
     wrapper::BlockWrapper,
 };
+use super::protocol::WorkScoreStructure;
```

Comment:
We already have a protocol import, merge

Reply:
Done